ts/Dashboards/Board.ts

```diff
@@ -52,10 +52,8 @@ const {
  * */
 
 /**
- * Creates a dashboard with components like charts, tables, and HTML.
+ * Class that represents a dashboard.
```

Comment:
Should this have an `@example` (or similar) like the [`Chart` class](https://github.com/highcharts/highcharts/blob/7ca7702a41215c6c9696f7c555df6cef1b4f384b/ts/Core/Chart/Chart.ts#L189-L199) has in Highcharts?